src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/GlobalSuppressions.cs

```diff
@@ -118,3 +118,11 @@
 [assembly: SuppressMessage("Globalization", "CA1308:Normalize strings to uppercase", Justification = "We require lowercase package names in telemetry so that the hashes are consistent", Scope = "member", Target = "~M:NuGet.PackageManagement.Telemetry.VSTelemetryServiceUtility.NormalizePackageId(System.String)~System.String")]
+[assembly: SuppressMessage("Usage", "VSTHRD010:Invoke single-threaded types on Main thread", Justification = "https://github.com/NuGet/Client.Engineering/issues/1321", Scope = "member", Target = "~M:NuGet.PackageManagement.VisualStudio.WebSiteProjectSystem.AddGacReference(System.String)")]
+[assembly: SuppressMessage("Usage", "VSTHRD010:Invoke single-threaded types on Main thread", Justification = "https://github.com/NuGet/Client.Engineering/issues/1321", Scope = "member", Target = "~M:NuGet.PackageManagement.VisualStudio.WebSiteProjectSystem.RemoveDTEReference(System.String)")]
+[assembly: SuppressMessage("Usage", "VSTHRD010:Invoke single-threaded types on Main thread", Justification = "https://github.com/NuGet/Client.Engineering/issues/1321", Scope = "member", Target = "~M:NuGet.PackageManagement.VisualStudio.EnvDTEProjectUtility.TryGetFolder(EnvDTE.ProjectItems,System.String,EnvDTE.ProjectItem@)~System.Boolean")]
+[assembly: SuppressMessage("Usage", "VSTHRD010:Invoke single-threaded types on Main thread", Justification = "https://github.com/NuGet/Client.Engineering/issues/1321", Scope = "member", Target = "~M:NuGet.PackageManagement.VisualStudio.EnvDTEProjectUtility.TryGetFile(EnvDTE.ProjectItems,System.String,EnvDTE.ProjectItem@)~System.Boolean")]
+[assembly: SuppressMessage("Usage", "VSTHRD010:Invoke single-threaded types on Main thread", Justification = "https://github.com/NuGet/Client.Engineering/issues/1321", Scope = "member", Target = "~M:NuGet.PackageManagement.VisualStudio.EnvDTEProjectUtility.TryGetNestedFile(EnvDTE.ProjectItems,System.String,EnvDTE.ProjectItem@)~System.Boolean")]
+[assembly: SuppressMessage("Usage", "VSTHRD010:Invoke single-threaded types on Main thread", Justification = "https://github.com/NuGet/Client.Engineering/issues/1321", Scope = "member", Target = "~M:NuGet.PackageManagement.VisualStudio.EnvDTEProjectUtility.GetReferencedProjects(EnvDTE.Project)~System.Collections.Generic.IList{EnvDTE.Project}")]
+[assembly: SuppressMessage("Usage", "VSTHRD010:Invoke single-threaded types on Main thread", Justification = "https://github.com/NuGet/Client.Engineering/issues/1321", Scope = "member", Target = "~M:NuGet.PackageManagement.VisualStudio.EnvDTEProjectUtility.GetWebsiteReferencedProjects(EnvDTE.Project)~System.Collections.Generic.IList{EnvDTE.Project}")]
+[assembly: SuppressMessage("Usage", "VSTHRD010:Invoke single-threaded types on Main thread", Justification = "https://github.com/NuGet/Client.Engineering/issues/1321", Scope = "member", Target = "~M:NuGet.PackageManagement.VisualStudio.EnvDTEProjectUtility.IsParentProjectExplicitlyUnsupported(EnvDTE.Project)~System.Boolean")]
```

Comment:
Please remove these suppressions. I just merged a PR that addressed them and removed the suppressions.

Reply:
See https://github.com/NuGet/NuGet.Client/pull/4435/files

Reply:
Removed.